lib/mix/tasks/nerves/new.ex

```diff
@@ -63,7 +63,7 @@ defmodule Mix.Tasks.Nerves.New do
   @nerves_pack_vsn "0.7.1"
   @toolshed_vsn "0.4.0"
 
-  @elixir_vsn "~> 1.11"
+  @elixir_requirement "~> 1.16"
```

Comment:
It seems we can't force 1.16 here and then also generate the version below based on the current version. Maybe we set this to `1.13` or get rid of the warning entirely?

Reply:
Good point. I like the warning a lot. I think this can be 1.13.